x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -392,17 +435,64 @@ func CalcTopicInitialRegret(regrets []alloraMath.Dec, epsilon alloraMath.Dec, pN
 		return alloraMath.ZeroDec(), err
 	}
 
-	minimumRegret := alloraMath.ZeroDec()
-	for i, regret := range regrets {
-		if i == 0 || regret.Lt(minimumRegret) {
-			minimumRegret = regret
-		}
+	// Calculate percentile
```

Comment:
it's a quantile not a percentile, it's not scaled to 100 percent it's between 0 and 1

Reply:
Didn't do anything about the feedback.  Great chap.  Unresolving.

Reply:
this actually was changed